src/content/en/updates/posts/2016/03/focus-start-point.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: updates/post
+title: "Removing Headaches from Focus Management"
+description: "The 'sequential focus navigation starting point' feature defines where we start to search for focusable elements for sequential focus navigation ([Tab] or [Shift-Tab]) when there is no focused area. It's especially helpful for accessibility features like skip links and managing focus in the document."
+published_on: 2016-03-15
+updated_on: 2016-03-15
+authors:
+  - robdodson
+tags:
+  - accessibility
+  - focus
+  - chrome50
+featured_image: /web/updates/images/2016/03/focus-start-point/focus-start-thumb.jpg
+---
+
+<p class="intro">The 'sequential focus navigation starting point' feature defines where we start to search for focusable elements for sequential focus navigation ([Tab] or [Shift-Tab]) when there is no focused area. It's especially helpful for accessibility features like "skip links" and managing focus in the document.</p>
+
+
+# Removing Headaches from Focus Management
+
+HTML provides us with a lot of built in support for dealing with keyboard interactions, which means it's pretty easy to write pages which can be used via the keyboard - whether a motor impairment prevents us from using a mouse, or we're just so efficient removing our hands from the keyboard wastes precious milliseconds.
+
+Keyboard handling revolves around focus, which determines where keyboard events will go in the page. There are a few situations in which, up till now, we've needed to do some extra work to make things work well for keyboard users. The `focus()` method allows us to manage focus by selectively choosing an element to focus in response to a user action. However, this best practice suffers from a lot of gotchas and requires some tricky JavaScript hackery to provide a baseline experience.
+
+While this technique isn't going to completely go away any time soon, in Chrome 50 it will be necessary in fewer cases thanks to the Sequential Focus Navigation Start Point. With this change, well-authored pages will automatically become more accessible without any need for extra manual focus management. Let's look at  an example.
+
+## Linking Within a Page
+Text heavy sites often interlink within the same page to help users quickly jump to important sections.
+
+{% highlight html %}
+<!-- Table of Contents -->
+<a href=”#recipes”>Recipes</a>
```

Comment:
You're using ” here and throughout, which aren't valid for quoting HTML attributes.